sw/airborne/boards/tawaki/chibios/v2.0/tawaki_2.0.cfg

```diff
@@ -19,7 +19,7 @@ HEADER
 
 #if !defined(STM32_HSECLK)
 #define STM32_HSECLK                8000000U
-#define STM32_HSE_BYPASS            TRUE
+#define STM32_HSE_BYPASS            
```

Comment:
What is the problem with this ?

Reply:
In chibios, only the definition is tested 
#if defined(STM32_HSE_BYPASS)

So it makes no sense to set it to True.